continuedev/continue
# Add a Delete Message button to the StepContainer

## Description

Add a Delete Message button to the StepContainer to allow the user to delete messages. This allows the user to shorten or clean up the context that is sent on request.

## Checklist


## Screenshots

<img width="307" alt="image" src="https://github.com/user-attachments/assets/b5c45d0c-a429-4518-86ed-0107a3e5bafa">

## Testing

1. Create a new message.
2. Hover over the StepContainer in a user message.
3. Click the Delete Message button which should be an X.
4. Expect the message to be deleted.
5. Write a new message referencing the deleted message and expect the LLM response to not know the content.
